aipl/ops/llm.py

```diff
@@ -21,15 +21,28 @@ def _parse_msg(s:str):
 
 # from the horse's mouth, 2023-05-30
 openai_pricing = {
-    "gpt-4-8k": 0.06,
+    "gpt-4": 0.06,
     "gpt-4-32k": 0.12,
     "gpt-3.5-turbo": 0.002,
     "ada": 0.0016,
     "babbage": 0.0024,
     "curie": 0.0120,
     "davinci": 0.1200
 }
+# base price covers the first 25 tokens, then it's the per-token price (2023-06-06)
+gooseai_pricing = {
+    "fairseq-13b": {
+        "base": 0.001250,
+        "token": 0.000036
+    },
+    "gpt-neo-20b": {
+        "base": 0.002650, 
+        "token": 0.000063
+    }
+}
 
+openai_models = set(openai_pricing.keys())
+gooseai_models = set(gooseai_pricing.keys())
```

Comment:
BTW, you don't have to make a separate set for these; `model in gooseai_pricing` will work just fine.

Reply:
hah! of course